Modules/OptionHolder.cs

```diff
@@ -824,6 +835,21 @@ public static void Load()
 
         CustomRoleManager.GetNormalOptions(Custom_RoleType.NeutralKilling).ForEach(r => r.SetupCustomOption());
 
+        TextOptionItem.Create(10000015, "RoleType.NeutralApocalypse", TabGroup.NeutralRoles)
+            .SetGameMode(CustomGameMode.Standard)
+            .SetColor(new Color32(127, 140, 141, byte.MaxValue));
+
+        TransformedNeutralApocalypseCanBeGuessed = BooleanOptionItem.Create(60024, "TNACanBeGuessed", false, TabGroup.NeutralRoles, false)
+            .SetGameMode(CustomGameMode.Standard)
+            .SetHeader(true);
+        CustomRoles.Baker.GetStaticRoleClass().SetupCustomOption();
+
+        CustomRoles.Berserker.GetStaticRoleClass().SetupCustomOption();
+
+        CustomRoles.PlagueBearer.GetStaticRoleClass().SetupCustomOption();
+
+        CustomRoles.SoulCollector.GetStaticRoleClass().SetupCustomOption();
```

Comment:
Btw, these roles can be automated via `CustomRoleManager.GetNormalOptions(Custom_RoleType.NeutralApocalypse).ForEach(r => r.SetupCustomOption());`

Reply:
oh yeah I tried that, but plaguebearer didn't show up in it, so I reverted it

Reply:
but then again that was before the Custom_RoleType stuff so maybe that will fix it, I'll try it